app/Http/Controllers/FaqController.php

```diff
@@ -7,12 +7,12 @@
 
 class FaqController extends Controller
 {
-    public function index(){
+    public function index() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->